components/match2/wikis/overwatch/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,66 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class OverwatchMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = WikiCopyPaste.Indent
+
+--returns the Code for a Match, depending on the input
+---@param bestof integer
+---@param mode string
+---@param index integer
+---@param opponents integer
+---@param args table
+---@return string
+function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
+	local showScore = Logic.nilOr(Logic.readBool(args.score), bestof == 0)
+	local opponent = WikiCopyPaste.getOpponent(mode, showScore)
+
+
+	local lines = Array.extendWith({},
+		'{{Match',
+		index == 1 and (INDENT .. '|bestof=' .. (bestof ~= 0 and bestof or '')) or nil,
+		Logic.readBool(args.needsWinner) and (INDENT .. '|winner=') or nil,
+		INDENT .. '|date=',
+		Array.map(Array.range(1, opponents), function(opponentIndex)
+			return INDENT .. '|opponent' .. opponentIndex .. '=' .. opponent
+		end),
+		Logic.readBool(args.streams) and (INDENT .. '|twitch=|youtube=|vod=') or nil,
+		bestof ~= 0 and Array.map(Array.range(1, bestof), function(mapIndex)
+			return INDENT .. '|map' .. mapIndex .. '={{Map|map=|mode=|score1=|score2=|winner=}}'
+		end) or nil,
+		INDENT .. '}}'
+	)
+
+	return table.concat(lines, '\n')
+end
+
+---@param template string
+---@param id string
+---@param modus string
+---@param args table
+---@return string
+---@return table
+function WikiCopyPaste.getStart(template, id, modus, args)
+	args.namedMatchParams = false
```

Comment:
this wanted?
if so you could just set it in the form for matchlist copy pasta
(the param doesn't have any impact on singlematch/bracket copy pasta)

if this is not wanted or set via the matchlist copy pasta form you can kick the entire function

Reply:
what exactly is the namedMatchParams? What does it do in relation to the other stuff?

Reply:
for matchlists it makes it
```
|{{Match|...}}
|{{Match|...}}
```
instead of
```
|M1={{Match|...}}
|M2={{Match|...}}
```

Reply:
Going to kick in favor of the second option. Removing the method seems to have had no issues with the form in testing